### ISSUE TITLE
Installation fails for py-pandoc 2.7.2

### ISSUE THREAD

**Issue by xuhcc** [kiwi0fruit/py-pandoc#4]
Versions:
- python 3.9.0
- pip 20.2.3

Here's the output
```
Building wheels for collected packages: py-pandoc
  Building wheel for py-pandoc (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: ./venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-4hfi61__/py-pandoc/setup.py'"'"'; __file__='"'"'/tmp/pip-install-4hfi61__/py-pandoc/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-gbyv8r17
       cwd: /tmp/pip-install-4hfi61__/py-pandoc/
  Complete output (5 lines):
  running bdist_wheel
  running build
  installing to build/bdist.linux-x86_64/wheel
  running install
  error: [Errno 2] No such file or directory: 'pandoc-2.7.2-0.tar.bz2'
  ----------------------------------------
  ERROR: Failed building wheel for py-pandoc
  Running setup.py clean for py-pandoc
Failed to build py-pandoc
```

Any ideas how to fix that?
The latest version of the package (2.9.2.1.2) builds correctly but I need 2.7.2 specifically (because newer pandoc versions have a bug that breaks my application).

**Comment by kiwi0fruit** [comment]
This is a won't fix for now as only 2.9.2.1.2 and 2.9.2.0.2 are patched to work with newer versions of pip. You can try downgrading to earlier version of pip. IIRC pip 20.0.2 worked fine and current version of pip doesn't work.

It can be fixed upstream - all is needed to be done it to fork current main and patch all versions and hashes for 2.7.2 values.

**Comment by kiwi0fruit** [comment]
As it's so easy to patch I decided to implement it: https://github.com/kiwi0fruit/py-pandoc/pull/5 . Now I'm waiting for https://travis-ci.org/github/kiwi0fruit/py-pandoc/builds/749005324 to complete.

**Comment by xuhcc** [comment]
Thank you!
Will it be available on PyPI?

**Comment by kiwi0fruit** [comment]
Yes it would be available.  But only after this CI https://travis-ci.org/github/kiwi0fruit/py-pandoc/builds/749005324 would end. But it takes too long for some reason.

**Comment by kiwi0fruit** [comment]
https://pypi.org/project/py-pandoc/2.7.2.0.1/
